processing/src/main/java/org/apache/druid/math/expr/vector/VectorProcessors.java

```diff
@@ -242,6 +270,12 @@ public void processIndex(Object[] strings, long[] longs, boolean[] outputNulls,
     return (ExprVectorProcessor<T>) processor;
   }
 
+  /**
+   * Creates an {@link ExprVectorProcessor} for the 'isnull' function, that produces a "boolean" (long) typed output
+   * vector with values set to 1 if the input value was null or 0 if it was not null.
```

Comment:
am I correct in understanding that this creates a `long[]`?

Reply:
yes, will clarify javadocs